src/main/java/com/minecolonies/coremod/blocks/BlockBarracksTowerSubstitution.java

```diff
@@ -120,6 +120,20 @@ public IBlockState withMirror(@NotNull final IBlockState state, final Mirror mir
         return state.withRotation(mirrorIn.toRotation(state.getValue(FACING)));
     }
 
+    /**
+     * Used to determine ambient occlusion and culling when rebuilding chunks
+     * for render.
+     *
+     * @return true
+     */
+    //todo: remove once we no longer need to support this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)